aldryn_newsblog/templates/aldryn_newsblog/includes/meta.html

```diff
@@ -0,0 +1,51 @@
+{% load meta_extra %}
+{% if meta %}
+    {% autoescape off %}
+	{% if meta.description %}{% meta 'description' meta.description %}{% endif %}
```

Comment:
I used this template:
https://github.com/nephila/djangocms-page-meta/blob/develop/djangocms_page_meta/templates/djangocms_page_meta/meta.html

Perhaps @Nephila also want to review how to improve it.

PS: Thanks to @Nephila for his work!